plugins/calendar/cmd/argParser.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"errors"
+	"log"
+	"time"
+
+	"github.com/disiqueira/ultraslackbot/plugins/calendar/pkg"
+)
+
+const (
+	cmdList   = "list"
+	cmdAdd    = "add"
+	cmdRemove = "remove"
+	msgList   = cmdList
+	msgAdd    = cmdAdd + " DD/MM/YYYY TITLE [DESCRIPTION]"
+	msgRemove = cmdRemove + " TITLE"
+)
+
+type ArgParser struct {
+	Calendar pkg.Calendar
```

Comment:
The Calendar interface should be declared here, so you don't create a dependency between the packages.